js/private/pnpm_utils.bzl

```diff
@@ -27,6 +27,16 @@ def _pnpm_name(name, version):
     "Make a name/version pnpm-style name for a package name and version"
     return "%s/%s" % (name, version)
 
+def _parse_pnpm_name(pnpmName):
+    # Parse a name/version or @scope/name/version string and return
+    # a [name, version] list
+    segments = pnpmName.split("/")
+    if len(segments) not in [2, 3]:
+        fail("unexpected pnpm versioned name " + pnpmName)
+    version = segments.pop()
+    name = "/".join(segments)
+    return [name, version]
```

Comment:
nit: could be slightly cleaner with
```
segments = pnpmName.rsplit("/", 1)
if len(segments) != 2:
    fail("unexpected pnpm versioned name " + pnpmName)
return [segments[0], segments[1]]
```
but loosing the error handling of ensuring that it is a 2 or 3 segments input